### PR TITLE
Enable -static-stdlib on Linux (#2921)

### PR DESCRIPTION
**Pull request by keith**
https://bugs.swift.org/browse/SR-648 has been resolved, so we can enable
this flag on Linux

Cherry pick of https://github.com/apple/swift-package-manager/pull/2921

### CODE REVIEW

**Comment by neonichu** [comment]
@swift-ci please test

**Comment by rballard** [comment]
Thanks very much; I expect we will take this as a change for the first post-5.3 dot release, so I'm holding this briefly before we merge into the 5.3 branch.

**Comment by rballard** [comment]
We've shipped 5.3 and are about to start taking fixes for the next patch release. Let's hold this one until Monday, but I expect to approve then.

**Review by rballard — APPROVED** [review]
This is approved to merge now; thank you!

**Pull request merged**
